### PR TITLE
line plot dev#2

### PR DESCRIPTION
**Pull request by linxu123-git**

### CODE REVIEW

File: src/plots/LinePlot.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  title: 'LinePlot',
+  component: LinePlot,
+};
+
+export const Basic = () => <LinePlot
+  onPlotUpdate={action('state updated')}
+  data={[{
+    x: ['A','B','C','D','E'],
+    y: [2,4,3,5,8],
+    mode: 'lines',
```
Comment (dmfalke) [inline src/plots/LinePlot.stories.tsx:15]: The property `mode` is not allowed here. This should be set in the `LinePlot` component.

File: src/plots/LinePlot.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+  yLabel="bar"
+/>
+
+const x = ['A','B','C','D','E'];
+export const MultiVariate = () => <LinePlot
+  onPlotUpdate={action('state updated')}
+  data={[{
+    x,
+    y: [1,2,3,4,5],
+    mode: 'lines',
```
Comment (dmfalke) [inline src/plots/LinePlot.stories.tsx:32]: The property `mode` is not allowed here. This should be set in the `LinePlot` component.

File: src/plots/LinePlot.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    name: 'Variable A',
+    line: {
+      dash: 'dashdot',
+      width: 4
+    }
+  }, 
+  {
+    x,
+    y: [1,4,9,16,25],
+    mode: 'lines',
```
Comment (dmfalke) [inline src/plots/LinePlot.stories.tsx:42]: The property `mode` is not allowed here. This should be set in the `LinePlot` component.

File: src/plots/LinePlot.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+    name: 'Variable B',
+    line: {
+      dash: 'dot',
+      width: 4
+    }
+  }, 
+  {
+    x,
+    y: [3,9,4,16,10],
+    mode: 'lines',
```
Comment (dmfalke) [inline src/plots/LinePlot.stories.tsx:52]: The property `mode` is not allowed here. This should be set in the `LinePlot` component.

File: src/plots/LinePlot.tsx [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+    xaxis: {
+      title: xLabel
+    },
+    yaxis: {
+      title: yLabel
+    }
+  }
+  return (
+  <
+    PlotlyPlot {...plotlyProps} layout={layout} type="scatter"
```
Comment (dmfalke) [inline src/plots/LinePlot.tsx:22]: `mode` should be set here.

**Comment by dmfalke** [comment]
I was actually thinking that `mode="lines"` would be set directly in `LinePlot.tsx`. Here is a diff illustrating my thinking:

```diff
diff --git a/src/plots/LinePlot.stories.tsx b/src/plots/LinePlot.stories.tsx
index 612475f..acf032f 100644
--- a/src/plots/LinePlot.stories.tsx
+++ b/src/plots/LinePlot.stories.tsx
@@ -20,7 +20,6 @@ export const Basic = () => <LinePlot
   }]}
   xLabel="foo"
   yLabel="bar"
-  mode="lines"
 />
 
 const x = ['A','B','C','D','E'];
@@ -58,7 +57,6 @@ export const MultiVariate = () => <LinePlot
 ]}
   xLabel="foo"
   yLabel="bar"
-  mode= "lines"
 />
 
 
diff --git a/src/plots/LinePlot.tsx b/src/plots/LinePlot.tsx
index b008991..be3a2e8 100644
--- a/src/plots/LinePlot.tsx
+++ b/src/plots/LinePlot.tsx
@@ -5,11 +5,10 @@ import { PlotComponentProps } from "./Types";
 export interface Props extends PlotComponentProps<'name'|'x'|'y'|'line'> {
   xLabel: string;
   yLabel: string;
-  mode: string;
 }
 
 export default function LinePlot(props: Props) {
-  const { xLabel, yLabel, mode, ...plotlyProps } = props;
+  const { xLabel, yLabel, ...plotlyProps } = props;^M
   const layout = {
     xaxis: {
       title: xLabel
@@ -20,7 +19,7 @@ export default function LinePlot(props: Props) {
   }
   return (
   <
-    PlotlyPlot {...plotlyProps} layout={layout} type="scatter"
+    PlotlyPlot {...plotlyProps} layout={layout} type="scatter" mode="lines"^M
     />
   )
 }
```

What do you think of this?

**Comment by linxu123-git** [comment]
That makes more sense!  I will update that.

**Pull request merged**